### ISSUE TITLE
Think how to do stable releases with DOI. 

### ISSUE THREAD

**Issue by ibartomeus** [ibartomeus/traitbase#9]
Maybe F1000 research is a good idea? 

Or simply DOI's via Zenodo or figshare and links to the original webpage.

**Comment by KevCaz** [comment]
Are you referring to the data base, not the package, right?

**Comment by ibartomeus** [comment]
Yep, zenodo/figshare could be a simple option. I want to ensure that if traitbase disappears for whatever reason (i hope not) the data is saved and discoverable.
